invenio_pages/views.py

```diff
@@ -74,34 +70,34 @@ def render_page(path):
         abort(404)
 
     return render_template(
-        [page.template_name, current_app.config['PAGES_DEFAULT_TEMPLATE']],
-        page=page)
+        [page.template_name, current_app.config["PAGES_DEFAULT_TEMPLATE"]], page=page
+    )
 
 
 def handle_not_found(exception, **extra):
     """Custom blueprint exception handler."""
     assert isinstance(exception, NotFound)
 
-    page = Page.query.filter(db.or_(Page.url == request.path,
-                                    Page.url == request.path + "/")).first()
+    page = Page.query.filter(
+        db.or_(Page.url == request.path, Page.url == request.path + "/")
+    ).first()
     if page:
         _add_url_rule(page.url)
         return render_template(
-            [
-                page.template_name,
-                current_app.config['PAGES_DEFAULT_TEMPLATE']
-            ],
-            page=page
+            [page.template_name, current_app.config["PAGES_DEFAULT_TEMPLATE"]],
+            page=page,
         )
-    elif 'wrapped' in extra:
-        return extra['wrapped'](exception)
+    elif "wrapped" in extra:
+        return extra["wrapped"](exception)
     else:
         return exception
 
 
 def _add_url_rule(url_or_urls):
     """Register URL rule to application URL map."""
-    if isinstance(url_or_urls, six.string_types):
+    if isinstance(url_or_urls, str):
         url_or_urls = [url_or_urls]
```

Comment:
Why this change?

Reply:
Six is a Python 2 and 3 compatibility library, but we only support python 3 thus is not needed.